libs/queue/src/main.sw

```diff
@@ -0,0 +1,39 @@
+library queue;
```

Comment:
See `std::bytes` for a reference; this needs docs added.

Reply:
Added docs